src/main/java/com/toptrumps/cli/CommandLineController.java

```diff
@@ -91,13 +91,14 @@ private void startNewGame(int numberOfOpponents) {
         while (players.size() != 1) {
             // == GAME START UP ==
             roundNumber++;
+            Logger.getInstance().logToFileIfEnabled(roundNumber + "");
```

Comment:
Can you add some text to show that this is the round number? At the moment it just logs the number and it's not clear what this is